endpoints.json

```diff
@@ -65,5 +63,35 @@
         }
       ]
     }
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "Responds with the comment object posted by the client",
+    "queries": [],
+    "exampleResponse": {
+      "comment": {
+        "comment_id": 19,
+        "body": "Has anyone heard any more about this?",
+        "article_id": 5,
+        "author": "lurker",
+        "votes": 0,
+        "created_at": "2023-09-02T13:53:23.000Z"
+      }
+    }
+  },
+  "PATCH /api/articles/:article_id/comments": {
+    "description": "Responds with the article object updated by the client",
+    "queries": [],
+    "exampleResponse": {
+      "article": {
+        "article_id": 30,
+        "title": "Top national parks in Costa Rica",
+        "topic": "travel",
+        "author": "jungle_nomad",
+        "body": "The highlight of any trip to Latin America...",
+        "created_at": "2023-11-21T19:15:00.000Z",
+        "votes": 1,
+        "article_img_url": "https://www.pexels.com/photo/mountains-under-blue-sky-1624880/"
+      }
+    }
```

Comment:
Its great to see both of these added to the endpoints.json file, but dont forget to give them both an example body of what we expect them to send with these two requests